packages/editor/src/client/atoms/infoMessage.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {ReactNode} from 'react'
+
+export interface InfoMessageProps {
+  type?: ReactNode
+  message: ReactNode
+}
+
+export function InfoMessage({type, message}: InfoMessageProps) {
+  if (type === 'warning') {
+    type = '#fffaf2'
```

Comment:
I think this is not recommended, you are mutating the prop(param), I think you can have 
```
let msgBgColor = '#ffffff'
``` 
for example and assign values to it.

-------------
You can also have colors as enum, if doesn't exist somewhere else, like:
```
export enum MessageColor {
  warning = '#fffaf2'
  error = '#fff2f2'
}
```